python/lsst/ts/xml/tables/m1m3/fa_table.py

```diff
@@ -0,0 +1,2791 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.  This product includes
```

Comment:
We have been using the Vera C. Rubin Observatory version for over two years now. Please update this line.